.github/workflows/todo.yaml

```diff
@@ -0,0 +1,15 @@
+name: Update TODO backlinks
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: todo-backlinks
+      uses: j2kun/todo-backlinks@v0.0.2
```

Comment:
This is such a cool action, I've always wanted something like this! Would you be OK / like me to share with other team I know who are using GitHub issues heavily in code comments?

minor nit: use the hash instead of version tagging since github versioning is mutable

Reply:
Of course! I just hacked it together over the holiday for fun, so any improvements or suggestions are welcome. Will fix the hash and then merge